dss/__init__.py

```diff
@@ -17,10 +17,14 @@
 
 logging.basicConfig(level=logging.DEBUG)
 
+app = None
```

Comment:
why do these need to be globals?

can the request methods just get the current app object via `flask.current_app`?